bench/plutus-scripts-bench/src/Cardano/Benchmarking/ScriptAPI.hs

```diff
@@ -0,0 +1,17 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Cardano.Benchmarking.ScriptAPI
+  (BenchScript, psName, psScript, mkBenchScript)
+  where
+
+import           Prelude as Haskell (String)
+import           Cardano.Api (ScriptInAnyLang)
+
+data BenchScript
+  = BenchScript
+    { psName   :: String
+    , psScript :: ScriptInAnyLang
+    }
```

Comment:
I apologise for not raising this matter sooner, since it'd save work, but I can't fail to mention this either.

The choice of name here is a bit confusing -- if taken with a bit of a wider context "bench script" can mean different things in benchmarking automation ecosystem.

A bit more specificity, like, perhaps, but not necessarily, `PlutusBenchScript` or `PlutusWorkloadScript` would be really useful.